testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListFragment.kt

```diff
@@ -43,7 +43,7 @@ class MiniAppListFragment : BaseFragment(), MiniAppListener, OnSearchListener,
     private lateinit var binding: MiniAppListFragmentBinding
     private lateinit var miniAppListAdapter: MiniAppListAdapter
     private lateinit var searchView: SearchView
-    private val preloadMiniAppWindow by lazy { PreloadMiniAppWindow(context!!, this) }
+    private val preloadMiniAppWindow by lazy { PreloadMiniAppWindow(requireContext(), this) }
```

Comment:
It's the same but `context!!` gives better minor performance :)

Reply:
Yeah still the same context, just wanted to avoid any possibility of NPE.

Reply:
Why do you want to avoid NPE? `requireContext()` is throwing `IllegalStateException` so I think they are same in exception throwing.